<ui version="4.0" >
 <class>MIACExchangeSetupDialog</class>
 <widget class="QDialog" name="MIACExchangeSetupDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Настройки соединения с МИАЦ</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="lblAddress" >
     <property name="text" >
      <string>&amp;Адрес</string>
     </property>
     <property name="buddy" >
      <cstring>edtAddress</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="edtAddress" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblPostBoxName" >
     <property name="text" >
      <string>&amp;Папка</string>
     </property>
     <property name="buddy" >
      <cstring>edtPostBoxName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="edtPostBoxName" />
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>151</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QCheckBox" name="chkCompress" >
     <property name="text" >
      <string>Сжимать данные</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QCheckBox" name="chkSendByDefault" >
     <property name="text" >
      <string>По умолчанию передавать данные </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtAddress</tabstop>
  <tabstop>edtPostBoxName</tabstop>
  <tabstop>chkCompress</tabstop>
  <tabstop>chkSendByDefault</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MIACExchangeSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>270</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MIACExchangeSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>338</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
